app/tasks/kis.py

```
@@ -0,0 +1,737 @@
+                avg_price = float(stock.get('pchs_avg_pric', 0))
+                current_price = float(stock.get('now_pric2', 0))
+                qty = int(float(stock.get('ovrs_cblc_qty', 0)))
+                
+                stock_steps = []
+                
+                # 1. 분석
+                self.update_state(state='PROGRESS', meta={'status': f'{symbol} 분석 중...', 'current': index, 'total': total_count, 'percentage': int((index / total_count) * 100)})
+                try:
+                    await analyzer.analyze_stock_json(symbol)
+                    stock_steps.append({'step': '분석', 'result': {'success': True, 'message': '분석 완료'}})
+                except Exception as e:
```

Ruff (0.14.6)

Do not catch blind exception: `Exception`

(BLE001)